src/angular/components/send/add-edit.component.ts

```diff
@@ -51,7 +51,6 @@ export class AddEditComponent implements OnInit {
     expirationDateSelect: number = null;
```

Comment:
Can you remove these unrelated changes to this file?

Reply:
Done, accidentally reverted it earlier